packages/uikit/src/hooks/translation.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from '@tonkeeper/core/dist/entries/language';
 import React, { useContext } from 'react';
 
-export type Translation = (text: string) => string;
+export type Translation = (text: string, values?: Record<string, string>) => string;
```

Comment:
It's work for i18n in browser

but not working for `browser.i18n.getMessage,` in extension